packages/core/integration-tests/test/javascript.js

```diff
@@ -2060,6 +2060,115 @@ describe('javascript', function () {
     assert.deepEqual(await (await run(b)).default, [3, 3]);
   });
 
+  it('does not delete a child bundle, type change or async, of a dynamic import that gets internalized', async () => {
+    let b = await bundle(
+      path.join(__dirname, '/integration/internalize-no-orphan-child/index.js'),
+    );
+
+    assertBundles(b, [
+      {
+        name: 'index.js',
+        assets: [
+          'index.js',
+          'bar.js',
+          'foo.js',
+          'esmodule-helpers.js',
+          'bundle-url.js',
+          'cacheLoader.js',
+          'js-loader.js',
```

Comment:
AFAICT because the async-import in foo.js of bazz.js isn't internalized. So these helper functions to perform the dynamic import at runtime are still needed

Reply:
Oh I see